api/src/main/java/org/openmrs/module/registrationapp/model/RegisterPersonRelationshipWidget.java

```diff
@@ -0,0 +1,79 @@
+package org.openmrs.module.registrationapp.model;
+
+import org.codehaus.jackson.annotate.JsonProperty;
+
+public class RegisterPersonRelationshipWidget extends Widget{
+
+    private Config config = new Config();
+
+    public RegisterPersonRelationshipWidget() {
+        super("registrationapp", "field/registerPersonRelationship");
+    }
+
+    public Config getConfig() {
+        if (config == null) {
+            config = new Config();
+        }
+        return config;
+    }
+    public void setConfig(Config config) {
+        this.config = config;
+    }
+
+    public static class Config {
+        @JsonProperty
+        private String type;
+        @JsonProperty
+        private String relationshipType;
+
+        @JsonProperty
+        private String gender;
```

Comment:
this limits the people returns by gender, if specified?  (makes sense, as long as null is allowed, just clarifying)